tests/backstop/scenarios.js

```diff
@@ -1,8 +1,84 @@
 const storybookHost = require( './detect-storybook-host' );
 const rootURL = `${ storybookHost }iframe.html?id=`;
 const storybookStories = require( '../../.storybook/storybook-data' );
+const filePaths = require( '../../.storybook/main' );
+const glob = require( 'glob' );
+const fs = require( 'fs' );
+const parser = require( '@babel/parser' );
+const traverse = require( '@babel/traverse' ).default;
+const csf = require( '@componentdriven/csf' );
+const kebabCase = require( 'lodash/kebabCase' );
 
-module.exports = storybookStories.map( ( story ) => {
+const newBackstopTests = [];
+
+const storyFiles =
+	// remove relative parent ..
+	glob.sync( filePaths.stories[ 1 ].slice( 1 ) );
```

Comment:
Here I'm just using the second value from 
`stories: [ '../stories/**/*.stories.js', '../assets/js/**/*.stories.js' ],`

If I use  '../stories/**/*.stories.js' then I get an error
```
Error: Invalid kind '', must include alphanumeric characters
    at sanitizeSafe (/home/main/wp-local-docker-sites/sitekit-10up
    ```
    
  Is it a problem to leave this one out? As I don't think there are any CSF stories in `./stories/**/*.stories.js'`

Reply:
Is this a problem? I thought there will be no CSF stories in `./stories/**/*.stories.js'` ?

I'm happy to look into this more but it could be time consuming

Reply:
`ReportError` and `ReportZero` stories are defined with CSF but don't have any scenarios defined for them.

I would prefer not to have any special handling here of the paths. Instead, we should be able to pass the [`cwd` option](https://github.com/isaacs/node-glob#options) to `glob.sync` so that they are processed relative to the `.storybook` directory.

Reply:
:thinking:   But then the paths won't work with `fs.readFileSync(`, as they're still relative.

The output of `glob.sync( '../assets/js/**/*.stories.js', { cwd: './.storybook' } );` is an array of relative paths 
`[ " '../assets/js/components/adminbar/AdminBarApp.stories.js', ... ]`